modules/cli/src/main/scala/scala/cli/commands/InstallCompletions.scala

```diff
@@ -43,12 +44,19 @@ object InstallCompletions extends ScalaCommand[InstallCompletionsOptions] {
         }
       }
       .getOrElse {
-        System.err.println(
-          "Cannot determine current shell, pass the shell you use with --shell, like"
-        )
-        System.err.println(s"  $name install completions --shell zsh")
-        System.err.println(s"  $name install completions --shell bash")
-        sys.exit(1)
+        lazy val fallbackAction = () => {
+          System.err.println(
+            "Cannot determine current shell, pass the shell you use with --shell, like"
+          )
+          System.err.println(s"  $name install completions --shell zsh")
+          System.err.println(s"  $name install completions --shell bash")
+          sys.exit(1)
+        }
+        val msg = "Cannot determine current shell. Which would you like to use?"
+        interactive.chooseOne(
+          msg,
+          List("zsh", "bash")
+        ).getOrElse(fallbackAction())
```

Comment:
NIT Isn't that clearer?
```suggestion
        val msg = "Cannot determine current shell. Which would you like to use?"
        interactive.chooseOne(msg, List("zsh", "bash")).getOrElse {
          System.err.println(
            "Cannot determine current shell, pass the shell you use with --shell, like"
          )
          System.err.println(s"  $name install completions --shell zsh")
          System.err.println(s"  $name install completions --shell bash")
          sys.exit(1)
        }
```